gdc_client/query/index.py

```diff
@@ -154,9 +155,18 @@ def separate_small_files(self,
         On top of that, separate the small files by open and controlled access
         so that if a controlled grouping failed, you can handle it as the same
         edge case.
+
+        Args:
+            ids (list): a set of file UUIDs
+            chunk_size (int): the maximum allowed combined size of small files
+
+        Return:
+            list: a list of big file UUIDs
+            list: a list of lists of UUIDs. Each inner list representing a
+                group of small files
         """
 
-        bigs = []
+        bigs = set()
```

Comment:
The resulting return is two lists, bigs and smalls. The reason that potential_smalls is a set is for an implicit uniqueness check. Please convert `bigs` to a list at the return of this function.

Bonus points: You don't have to update the docstring if you change it to a list. :-)